verify-node-version.js

```diff
@@ -1,6 +1,12 @@
-const requiredVersion = require("fs").readFileSync(".nvmrc", { encoding: "utf8" }).trim()
+const requiredVersion = require("fs")
+  .readFileSync(".nvmrc", { encoding: "utf8" })
+  .trim()
 
-if (process.version !== requiredVersion) {
-  console.error(`[!] This project requires Node.js ${requiredVersion}, current version is ${process.version}`)
+if (process.version.split(".")[0] !== requiredVersion.split(".")[0]) {
```

Comment:
Updates our version check script to reflect what's specified in the `engines` field of the package.json. Currently we're only requiring `8.x.x`. Loosening this restriction allows us to use the latest version of the docker image which includes `yarn`. That removes the need for us to install `nvm` and download `node` or `yarn`.